sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -213,4 +225,29 @@ class SentryAppenderTest {
             })
         }
     }
+
+    @Test
+    fun `attaches breadcrumbs`() {
+        fixture = Fixture(minimumBreadcrumbLevel = Level.DEBUG, minimumEventLevel = WARN)
+        val utcTime = LocalDateTime.now(ZoneId.of("UTC"))
+
+        fixture.logger.debug("this should be a breadcrumb #1")
```

Comment:
One test case could make sure this doesn't end up in a crumb because min level is Info.

Actually a test case for default would be nice, and default would be Info or higher for Breadcrumb and Error or higher for event.